src/modules/aws.rs

```diff
@@ -47,19 +46,22 @@ fn get_aws_region_from_config(aws_profile: Option<&str>) -> Option<Region> {
     Some(region.to_string())
 }
 
-fn get_aws_profile_and_region() -> (Option<Profile>, Option<Region>) {
+fn get_aws_profile_and_region(contex: &Context) -> (Option<Profile>, Option<Region>) {
```

Comment:
Looks like there might be a typo here and in future uses of the parameter:
```suggestion
fn get_aws_profile_and_region(context: &Context) -> (Option<Profile>, Option<Region>) {
```

Reply:
I mistyped the parameter and auto-completion hid the typo from me. Fixed throughout the whole function.